code/clue.js

```diff
@@ -1,59 +1,351 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+const loader = document.createElement("div");
+loader.classList.add("clock");
+const handle = document.createElement("div");
+handle.classList.add("handle");
 
-const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
-  age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+const showLoader = (card) => {
+  loader.appendChild(handle);
+  card.appendChild(loader);
+};
```

Comment:
I think it is interesting to try to understand your code here regarding how you solved the loader. I'm not sure I am able to follow entirely what's happening in your code. But If I read the code correctly, I think that you created a new div in JS with createElement. And then you use appendChild in some way in a showLoader function, which you later on in the pickKiller-function you call the loader function. 
It's interesting to see that you solved the loader in a different way than I did, and I learned more by trying to read your code here 👍  
I made a loader-div directly in HTML on each card & specified it to visibility: hidden in CSS and then in the JS functions to pick killer etc, I used an if-statement and changed the style to visible using a timeout-function. So that could be an alternative way to approach this also.

Reply:
Definitely! From my perspective I just want to add that both approaches are doing the same thing, but as mentioned on Wednesday's lecture, it's better from efficiency point of view to add all HTML elements in HTML file and just manipulate their attributes in JS.
Nevertheless Emelie, don't consider my comment as "I don't like how you've done it" - it's great you learn new tools, this is the essence of being programmer, so great job :) Just remember that some tools are better